pkg/model/deployment.go

```diff
@@ -20,6 +20,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const DeploymentMeta_NotificationsKey = "DeploymentNotification"
```

Comment:
How about 

``` golang
const (
    MetadataKeyDeploymentNotification = "DeploymentNotification"
)

Reply:
Looks more like go code 👍  lets me adopt that 🙆‍♀️

Reply:
btw, in that case, should we move that to https://github.com/pipe-cd/pipe/blob/master/pkg/app/piped/metadatastore/store.go

Reply:
On the second thought, I think it's better to be a part of the deployment model since we use it mostly when we manipulate the model object.

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2799/commits/13616bbddcecd2434e515ee2f8aff517a4d84d95